cypress/fixtures/economyArticlesResponse.json

```diff
@@ -3,37 +3,32 @@
     {
       "id": 482,
       "title": "Biggest financial crisis ever",
-      "body": "Tips and trics how not to become poor",
-      "created_at": "2021-12-10T10:24:38.354Z",
-      "updated_at": "2022-01-13T10:24:38.354Z"
+      "teaser": "Tips and trics how not to become poor",
+      "published": "2021-12-10T10:24:38.354Z"
```

Comment:
Category is missing, this fixture doesn't match the api response,
example:
{"id"=>236,
    "title"=>"Covid-19 is a pandemic",
    "teaser"=>"This is my article body...",
    "published"=>"2022-01-23T16:32:50.881Z",
    "category"=>"Politics"}

Reply:
Fixed